pkg/okteto/kubetoken_client.go

```diff
@@ -98,5 +98,5 @@ func (c *kubeTokenClient) CheckService(baseURL, namespace string) error {
 		return fmt.Errorf("CheckService %w: %s", errKubetokenNotAvailable, baseURL)
 	}
 
-	return nil
+	return resp.Body.Close()
```

Comment:
Same as above. We should be ignoring the error but logging it in case we need that information

Reply:
changed the code, let me know if 👍🏻